tic_tac_toe/utils.py

```diff
@@ -49,7 +49,7 @@ def usr_input(player):
 
             return row, column
         except (ValueError, IndexError):
-            print('Incorrect input. Try again. ')
+            print('Incorrect input. Try again.(Your input needs to be in the format <number><space> <number> i.e "2 2" for the middle of the board) ')
```

Comment:
```suggestion
            print('Incorrect input. Try again.\n'\
                  '(Your input needs to be in the format <number><space><number> '
                  'i.e "2 2" for the middle of the board) ')
```
I think it will look [better](https://imgur.com/JGBmWcy).